docs/no_numerical_exp_report.tex

```diff
@@ -72,17 +76,86 @@ \section{Method}
 
 
 \section{Configuration}
-The user defines prototypes to be deployed for fuel facilities,
-and reactor deployment scheme.  The reactor deployment causes the demand of fuel
+The user defines prototypes to be deployed for fuel facilities, and the 
+demand equation.  The reactor deployment causes the demand of fuel
 which triggers fuel facility deployment. The detailed input file XML input schema
 is shown in Appendix A. 
 
 \section{Algorithm Flow}
 
-The algorithm, upon entering, creates a supply chain with the fuel facilities and
-the reactor. Then, at every timestep it calculates the expected demand from each fuel cycle
-facility and makes decisions to deploy or decommission. As a reference, the time step
-execution for \Cyclus is illustrated in figure \ref{diag:time}.
+The algorithm is an \texttt{Institution}, which is a \Cyclus agent type
+that deploys and decommissions facilities. 
+
+The algorithm predicts the demand by looking at the previous demand and supply values.
```

Comment:
Perhaps: The algorithm predicts the demand of a commodity by looking at its previous demand and supply values.